proto/osmosis/concentrated-liquidity/query.proto

```diff
@@ -209,3 +217,19 @@ message PoolAccumulatorRewardsResponse {
   repeated UptimeTracker uptime_growth_global = 2
       [ (gogoproto.moretags) = "yaml:\"list\"", (gogoproto.nullable) = false ];
 }
+
+// ===================== QueryTickAccumulatorTrackers
+message TickAccumulatorTrackersRequest {
+  uint64 pool_id = 1 [ (gogoproto.moretags) = "yaml:\"pool_id\"" ];
+  int64 tick_index = 2 [ (gogoproto.moretags) = "yaml:\"tick_index\"" ];
+}
+
+message TickAccumulatorTrackersResponse {
+  repeated cosmos.base.v1beta1.DecCoin
+      spread_reward_growth_opposite_direction_of_last_traversal = 1 [
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.DecCoins",
+    (gogoproto.nullable) = false
+  ];
+  repeated UptimeTracker uptime_trackers = 2
+      [ (gogoproto.moretags) = "yaml:\"list\"", (gogoproto.nullable) = false ];
```

Comment:
I think we should use this instead of list here, (I dont see any reason to use list here)
```suggestion
      [ (gogoproto.moretags) = "yaml:\"uptime_trackers\"", (gogoproto.nullable) = false ];
```